tavern/schemas/extensions.py

```diff
@@ -13,6 +14,23 @@
 logger = logging.getLogger(__name__)
 
 
+def vaidate_last(func):
+    """Decorator to execute custom validator after all nested validation will
+    be passed."""
+    @functools.wraps(func)
+    def wrapper(value, rule_obj, path):
+        schema = rule_obj.schema
+        if schema is not None:
+            schema = schema.copy()
+            del schema['func']
+            c = core.Core(source_data=value, schema_data=schema)
+            c.validate(raise_exception=True)
```

Comment:
Can you add some documentation to what this is doing? I get that it does some extra validation on the input but I don't understand why or how